documentation/source/users/rmg/installation/anacondaDeveloper.rst

```diff
@@ -47,15 +47,6 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
    to install this is to simply run one of the commands in the terminal, e.g. ``git``. The terminal will then prompt you on
    whether or not you would like to install the Command Line Tools.
 
-   For MacOS users only, download and install the latest macOS julia from here: <https://julialang.org/downloads/>. Then add julia to PATH by running the following command replacing 1.6 with the first two numbers in the Julia version designation (ex: 1.6.2->1.6, 1.1.1->1.1) ::
```

Comment:
Are we sure that the Julia packaged within conda works on MacOS now? I remember @mjohnson541 said that the conda installation of Julia on MacOS didn't respect the environment and that was why this was recommended.

Reply:
I tried a fresh installation just now, and I can confirm that the conda Julia is still not working for MacOS still. Here's the error trace I got:

```
(rmg_env_20230418) hwpang@Hao-Weis-MacBook-Pro RMG-Py % julia -e 'using Pkg; Pkg.add(PackageSpec(name="ReactionMechanismSimulator",rev="main")); using ReactionMechanismSimulator;'
zsh: killed     julia -e 
```